app/uk/gov/hmrc/rasapi/controllers/FileProcessingController.scala

```diff
@@ -47,15 +47,15 @@ class FileProcessingController @Inject()(
         case (Some(apiVersion), Some(callbackData)) =>
           callbackData.status match {
             case STATUS_AVAILABLE =>
-              Logger.warn(s"[FileProcessingController] [statusCallback] Callback request received with status available: file processing " +
+              Logger.info(s"[FileProcessingController][statusCallback] Callback request received with status available: file processing " +
                 s"started for userId ($userId).")
               if (Try(fileProcessingService.processFile(userId, callbackData, apiVersion)).isFailure) {
                 sessionCacheService.updateFileSession(userId, callbackData, None, None)
               }
-            case STATUS_ERROR => Logger.error(s"[FileProcessingController] [statusCallback] There is a problem with the " +
+            case STATUS_ERROR => Logger.error(s"[FileProcessingController][statusCallback] There is a problem with the " +
               s"file for userId ($userId) ERROR (${callbackData.fileId}), the status is: ${callbackData.status} and the reason is: ${callbackData.reason.get}")
               sessionCacheService.updateFileSession(userId, callbackData, None, None)
-            case _ => Logger.warn(s"There is a problem with the file (${callbackData.fileId}) for userId ($userId), the status is:" +
+            case _ => Logger.error(s"[FileProcessingController][statusCallback] There is a problem with the file (${callbackData.fileId}) for userId ($userId), the status is:" +
               s" ${callbackData.status}")
           }
           Future(Ok(""))
```

Comment:
Line 62, We return a BadRequest, there is no log here. The causes will either be an incorrect version being passed in or a lack of valid json. The 'withValidJson' function had warn logs when it passes a none but the None passed on line 44 for the version match doesn't have a corresponding one.